aries-site/src/layouts/content/InPageNavigation.js

```diff
@@ -61,6 +63,8 @@ export const InPageNavigation = ({ headings }) => {
   // align "Jump to section" with page title at start
   const marginTop = `${large + medium}px`;
 
+  const { contentHistory } = useContext(ViewContext) || undefined;
```

Comment:
Is the `|| undefined` needed? Seems like ViewContext is always defined since it wraps the whole app?